written by Mike Frysinger

https://sourceforge.net/tracker/?func=detail&aid=3132056&group_id=5616&atid=305616

nommu systems cannot fork() as the hardware cannot support
it. irattach uses it as a minor optimization, but it isnt
necessary for correct functioning of the utility. so add a
NO_FORK define so we nommu peeps can do CFLAGS="... -DNO_FORK=1
..." and use it in our embedded systems.

--- a/irattach/irattach.c
+++ b/irattach/irattach.c
@@ -397,7 +397,11 @@
 					after_names[i]);
 				/* Create a new instance for this other
 				 * interface */
+#ifdef NO_FORK
+				pid = -1;
+#else
 				pid = fork();
+#endif
 				/* If in the child */
 				if(!pid) {
 					/* Get the interface name */
